### DISCUSSION TITLE
Welcome to Tekinged Discussions!

### DISCUSSION THREAD

**Discussion by johnbent** [tekinged/missing#57] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by s-o-u-a-n-g** [comment]
It is rude to call someone who is bedridden “dengarech”, so instead we say “melanges buadel”. 

Looking up toward the beams. 

A demal el rubak a kmal chuodel e mechitechut me ng di melanges buadel kung. 

souang

**Reply by johnbent** [reply]
Thanks @s-o-u-a-n-g ! Great stuff as always from you. I'm going to turn this into an Issue. :)
